ui/src/settings.vue

```diff
@@ -835,7 +960,13 @@ export default {
                 });
             }
         },
-        queryUser : function() {
+        userEditJSON: function() {
```

Comment:
Please change the function names to reflect what these functions actually do. Maybe something like "convertRawJSONtoUserEdit"?

Reply:
It is the model in watcher which updates the UI (userEdit) from the JSON editor (userEditJSON). 
How about I change the variable to userEditJSONtoUI?